submodules/ir_blaster.py

```diff
@@ -0,0 +1,16 @@
+
+class IRBlasterController:
+    
+    def __init__(self, config):
+        self.command_port = config['command_port']
+        self.supported_commands = config['supported_commands']
+
+    def send_command(self, command_data):
+        command = command_data.get('command')
+        
+        if command not in self.supported_commands:
+            raise ValueError(f"Unsupported command: {command}")
+
+        # simulate sending command to IR Blaster
+        print(f"Sending IR command '{command}' to port {self.command_port}")
```

Comment:
how will we get this class though normally it's `self.sendKey(self, keycode:dict, delay:int=1, repeat:int=1, randomRepeat:int=0):` because the class is `self.commonRemote = self.dut.remoteController`